src/BloomExe/Workspace/WorkspaceView.cs

```diff
@@ -188,14 +184,44 @@ void AdjustToolPanelLocation(bool allowNarrowing)
 		/// Adjust the tool panel location when the chosen localization changes.
 		/// See https://jira.sil.org/browse/BL-1212 for what can happen if we
 		/// don't adjust.  At the moment, we only widen, we never narrow the
-		/// area allotted to the tab buttons.
+		/// overall area allotted to the tab buttons.  Button widths adjust
+		/// themselves automatically to their Text width.  There doesn't seem
+		/// to be a built-in mechanism to limit the width to a given maximum
+		/// so we implement such an operation ourselves.
 		/// </summary>
 		void HandleTabTextChanged(object sender, EventArgs e)
 		{
+			var btn = sender as Messir.Windows.Forms.TabStripButton;
+			if (btn != null)
+			{
+				const string kEllipsis = "\u2026";
+				// preserve the original string as the tooltip
+				if (!btn.Text.EndsWith(kEllipsis))
+					btn.ToolTipText = btn.Text;
```

Comment:
They did before because the AutoToolTip flag is set.  The ToolTipText wasn't being set, so the tooltip showed the Text value.